src/framework/preferences/html/PrefsEditorTemplate-layout.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-toc"></span><span class="flc-prefsEditor-toc-label heading-text"></span></h2>
-
+<p class="flc-prefsEditor-toc-choice-label"></p>
```

Comment:
Are these really descriptions and not labels? If they are really descriptions, the class should probably be named something related to that instead of label. Same throughout.